### PR TITLE
Adding compliance operator

### PR DESCRIPTION
**Pull request by varad-ahirwadkar**
This role has dependency on two roles:
- Cluster health check
- OCP cli plugins

Signed-off-by: Varad Ahirwadkar <varad.ahirwadkar@ibm.com>

### CODE REVIEW

File: playbooks/roles/ocp-compliance/README.md [hunk 1]
```diff
@@ -0,0 +1,56 @@
...
+The cluster is in a known good state, without any errors.
+
+Role Variables
+--------------
+
+Role Variables
+--------------
+| Variable                       | Required | Default     | Comments                                       |
+|--------------------------------|----------|-------------|------------------------------------------------|
+| compliance_enabled             | yes      |    false    | Set it to true to run this playbook            |
```
Comment (yussufsh) [inline playbooks/roles/ocp-compliance/README.md:23]: Same comment as previous PR... IMO cannot be Required and default needs to be defined.

Comment (varad-ahirwadkar) [reply playbooks/roles/ocp-compliance/README.md]: I have made the changes

**Review by pravin-dsilva — APPROVED** [review]

**Pull request merged**